config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
What does a Validation look like? If we are going to make Validations (rather than just Errors), then I would be happy to promote this endpoint to a first-class controller. What would you do with a Validation, besides report it back to the user?

I would rather see `GET /manifests` route to `manifests#search` rather than have a whole controller.

But "standard" routing structure do you mean CRUD-like Rails routing? I consider that one standard among others.